app/assets/javascripts/users/promote.js.coffee

```diff
@@ -0,0 +1,6 @@
+$(document).ready ->
+  $ ->
```

Comment:
I found that the autocomplete would not work without it. [Here](https://jqueryui.com/autocomplete/) you can see the javascript code for a working autocomplete. When converted into coffee script it looks similar to what is shown here.